src/sangeeta/NumberIdentifierByWhileLoop.java

```diff
@@ -0,0 +1,61 @@
+package sangeeta;
+
+public class NumberIdentifierByWhileLoop {
+	
+	void printEvenNumbers(int startIndex, int endIndex) {
+		int index = startIndex;
```

Comment:
you don’t require to create extra variable, you can handle this without **index**

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files